src/services/GraphQl/subscription/index.js

```diff
@@ -0,0 +1,26 @@
+import gql from "graphql-tag";
+
+export const GET_PACKAGE_INFORMATION_SUBSCRIPTION = gql`
+subscription getPackageInformation{
+  packages(where: {package_id: {_eq: "hola"}}) {
```

Comment:
please erase this test and make this dynamic

Reply:
Also please change as I changed the key here:
![image](https://user-images.githubusercontent.com/36519478/135973537-f85a4f40-fdb3-48b0-825e-7183cce84492.png)
```suggestion
  packages(where: {package_code: {_eq: "hola"}}) {
```